.github/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
is this a good idea? obviously we like OSS but we don't want to accidentally give ourselves a legal obligation to publish the source to internal stuff before we're ready